### PR TITLE
remove useless conversions

### PR DESCRIPTION
**Pull request by danieleades**

### CODE REVIEW

File: src/cargo/core/shell.rs [hunk 1]
```diff
@@ -457,7 +457,7 @@ mod imp {
             let mut winsize: libc::winsize = mem::zeroed();
             // The .into() here is needed for FreeBSD which defines TIOCGWINSZ
             // as c_uint but ioctl wants c_ulong.
-            if libc::ioctl(libc::STDERR_FILENO, libc::TIOCGWINSZ.into(), &mut winsize) < 0 {
+            if libc::ioctl(libc::STDERR_FILENO, libc::TIOCGWINSZ, &mut winsize) < 0 {
```
Comment (joshtriplett) [inline src/cargo/core/shell.rs:460]: Note the comment right above this, which explains the `.into()`: it's platform-specific.

Comment (danieleades) [reply src/cargo/core/shell.rs:460]: Aha! I see

**Comment by danieleades** [comment]
@joshtriplett i think this should be ready for review.

whopping great change that it is!

**Pull request merged**